typed-racket-doc/typed-racket/scribblings/reference/optimization.scrbl

```diff
@@ -27,3 +27,13 @@ is insufficiently powerful to access @racketmodname[racket/unsafe/ops],
 for example when executing in a sandbox (see @secref["Sandboxed_Evaluation"
 #:doc '(lib "scribblings/reference/reference.scrbl")]). This prevents untrusted
 code from accessing these operations by exploiting errors in the type system.
+
+
+@section{Contract Optimization}
+
+Typed Racket generates contracts for its exports to protect them against
+untyped code.
+By default, these contracts do not check that typed code obeys the types.
+If you want to generate contracts that check both sides equally (for analysis,
+for teaching, etc.) then set the environment variable
+@envvar{PLT_TR_NO_CONTRACT_OPTIMIZE} to any value and recompile.
```

Comment:
I think we might need to be more specific here. Simply recompiling the project that depends on TR won't take effect, as TR needs to be recompiled as well. IOW, i.e. `raco setup typed-racket typed <your-collection>`.

Reply:
Why is this comment hidden? Should I reword things or is it ok?

I agree that it'd be good to be more specific ... we could easily say `recompile both Typed Racket and your code` ... but all environment variables have the same problem and maybe this page of the docs needs a bigger change.

Reply:
I wanted to run some experiment to further verify my comment so I hid it. And I agree that this is a common problem regarding environment variables.  maybe we need to have an entry (e.g. via deftech) for `recompile`.